src/java/arjdbc/jdbc/RubyJdbcConnection.java

```diff
@@ -3752,6 +3752,7 @@ protected static RubyString cachedString(final ThreadContext context, final Stri
         public final int type;
 
         private final String label;
+        private IRubyObject rubyType = null;
```

Comment:
@rdubya I assume this is accessed from Ruby somewhere?  I don't see any consumers.